lib/views/bidding_slack_view.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Aprb.Views.BiddingSlackView do
+  import ViewHelper
+
+  def render(event) do
+    artwork_data = fetch_sale_artwork(event["lotId"])
+    %{
+      text: ":gavel: #{event["type"]} on #{artwork_data[:permalink]}",
+      attachments: "[{
+                      \"fields\": [
+                        {
+                          \"title\": \"Amount\",
+                          \"value\": \"#{format_price((event["amountCents"] || 0) / 100)}\",
```

Comment:
oddly enough, we were multiplying the price by 100 in `format_price` anyways! All I have to do is to remove `/100` here and `*100` in format price. `APRB` code still feels hackthonish here and there heheh